src/main/java/core/inventory/items/Sword.java

```diff
@@ -22,12 +22,14 @@
     private final int range;
     private final int damage;
     public UUID id;
+    static final int SWORD_DAMAGE = 3;
+    static final int SWORD_RANGE = 2;
 
     public Sword(int level) {
         this.level = level;
         this.size = this.level;
-        this.range = this.level * 2;
-        this.damage = this.level * 3;
+        this.range = this.level * SWORD_RANGE;
+        this.damage = this.level * SWORD_DAMAGE;
```

Comment:
see comment in Dagger constructor

Reply:
also, if `range` and `damage` weren't `final` variables, the damage + range setting should happen in the setter for level, so that they are always consistend with level

(just in case)

Reply:
Could we instead get rid of the setter for level since I don't think anything uses it (as the weapon gets instantiated with level as a parameter anyway).

Also by speed of characters in the Character class are you referring to setVelocity? Can't seem to find any hardcoded numbers.

Reply:
level system is pretty important I think it's worth keeping